tests/zero_temperature/statistics

```diff
@@ -3,13 +3,12 @@
 # 3: Number of mesh cells
 # 4: Number of Stokes degrees of freedom
 # 5: Number of temperature degrees of freedom
-# 6: Iterations for temperature solver
-# 7: Iterations for Stokes solver
-# 8: Velocity iterations in Stokes preconditioner
-# 9: Schur complement iterations in Stokes preconditioner
-# 10: Time step size (seconds)
-# 11: Minimal temperature (K)
-# 12: Average temperature (K)
-# 13: Maximal temperature (K)
-0 0.000000000000e+00 1024 9539 4225 0 37 50  8 1.000000000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00 
-1 1.000000000000e+00 1024 9539 4225 0 29 30 30 3.141683944776e+15 0.00000000e+00 0.00000000e+00 0.00000000e+00 
+# 6: Iterations for Stokes solver
+# 7: Velocity iterations in Stokes preconditioner
```

Comment:
I though about this a while: your patch suppresses the output of the number of iterations. That's at first surprising, but it's also not wrong -- no solve was performed. But whether the solve step is skipped may vary from time step to time step, so it may be that for some rows of the file, the column is empty (indicated by `""`) and for others has a numeric value (which may be zero).

Thinking about it, I'm ok with this outcome, but I wanted to bring it up for discussion in case someone feels different.